third_party/proto/tendermint/abci/types/types.proto

```diff
@@ -60,11 +62,10 @@ message RequestSetOption {
 }
 
 message RequestInitChain {
-  google.protobuf.Timestamp time = 1
```

Comment:
fixed a lot of these here: https://github.com/tendermint/tendermint/pull/4386 this will be in the next release. The release will be coming in the coming days